package.json

```diff
@@ -4,10 +4,13 @@
   "description": "An HTMLElement extension that provides a simple setup for Web Components using React.",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "test": "npm run lint && npm run build",
+    "test": "npm run lint && jest && npm run build",
```

Comment:
Should `jest` be run before or after `build`? I'm partial to after, since it keeps the typscript and eslint linting results together.

Reply:
I'm ok with that! I can add this as the last step.